test/arrays/slices/localSliceDRSlice.chpl

```diff
@@ -0,0 +1,39 @@
+module Spam {
+    use BlockDist;
+    use IO;
+    use IO.FormattedIO;
+    use Reflection;
+
+    proc sliceAndDice(fn: string, line: int, ref arr) throws {
+        for l in arr.targetLocales do on l {
+          for d in arr.localSubdomains() {
+//            if !domainDistIsLayout(arr.domain) {
```

Comment:
Can this comment be removed? Or if it was useful in testing maybe guard behind a config or explain what it was for

Reply:
It can.  And you're reminding me that I also changed John's code to see what would happen if ranges were used instead of a domain (where that test case could be worth preserving, but shouldn't be the only one).